apps/sr-frontend/src/app/App.tsx

```diff
@@ -28,6 +34,16 @@ export default function App() {
               <Stack gap={1} pt={1}>
                 <CharacterSelector />
                 <Character />
+                <Container>
```

Comment:
@EuEric Can you wrap the Accordion in a `<CardThemed/>`, and move this entire thing as a separate component in `/RelicInventory.tsx`?

I would also recommend moving this Relic Inventory element below the `<RelicEditor/>` for slightly better flow.

Reply:
Sure, no problem, will work on it starting next week

Reply:
Moved whole Accordion wrapped in CardThemed inside RelivInventory and placed RelicEditor ahead